operations/iuf/workflows/configuration.md

```diff
@@ -97,6 +110,9 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
        ./site_vars.yaml
        ```
 
+Until Jira CRAYSAT-1900 is resolved, the sat tool will not automatically replace the variables `system-name` and `site-domain` with the values defined in `site_vars.yaml` in the `${ADMIN_DIR}/bootprep/compute-and-uan-bootprep.yaml` file. Therefore, they must
```

Comment:
Please remove this internal Jira reference. We should avoid this in the public docs.
Maybe start this line with..

SAT will not automatically replace..

Reply:
Dirty laundry hidden. We do expect the fix to CRAYSAT-1900 imminently, so my plan was never to leave this comment in for long. It was mostly to reassure the installers.